core/src/main/java/com/opensymphony/xwork2/ognl/OgnlValueStackFactory.java

```diff
@@ -125,11 +125,17 @@ protected boolean containerAllowsStaticMethodAccess() {
 
     /**
      * Retrieve allowStaticFieldAccess state from the container (allows for lazy fetching)
+     * Note: Defaults to true when no state is defined in the container (same default behaviour as OgnlUtil)
      * 
      * @return
      */
     protected boolean containerAllowsStaticFieldAccess() {
-        return BooleanUtils.toBoolean(container.getInstance(String.class, StrutsConstants.STRUTS_ALLOW_STATIC_FIELD_ACCESS));
```

Comment:
I investigated why `containerAllowsStaticMethodAccess()` doesn't need such fix and found [this in default.properties](https://github.com/apache/struts/blob/9a14323a6a6414f845f240c8463503e0c940cec6/core/src/main/resources/org/apache/struts2/default.properties#L207)! I think we should keep same conduct as static methods for static fields.

Reply:
Maybe we should drop that file and move everything into `struts-default.xml`?